### ISSUE TITLE
Fighting distance is weird in this simulator

### ISSUE THREAD

**Issue by utensil** [theRobinator/League-of-Legends-Simulator#2]
Hi!

It's great to see someone writing an LOL simulator. I read through the the code and the implementation is quite straight forward. 

But there's something weird when champions are fighting.

You seem to choose a distance that is Math.max(champ.range, enemy.range) to start fighting, but if so, for two champions with different attack range, one will fail to attack the other...And your champions can't move it seems...

**Comment by theRobinator** [comment]
You're right, range mechanics haven't been implemented yet. The next planned patch is an object for a champion's actions, which will add ranges to actions and put a delay on the time that it takes to execute the action based on the champion's move speed and range combined.
